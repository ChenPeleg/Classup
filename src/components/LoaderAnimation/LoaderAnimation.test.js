import React from "react";
import { shallow } from "enzyme";
import Loader from "./LoaderAnimation";

const loader = shallow(<Loader />);
describe("<Loader anination>", () => {
  it("renders correctly", () => {
    expect(loader).toMatchSnapshot();
  });
});
