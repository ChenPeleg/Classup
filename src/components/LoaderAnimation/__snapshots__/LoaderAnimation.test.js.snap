// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader anination> renders correctly 1`] = `ShallowWrapper {}`;
